resume/src/components/sections/Sections.css

```diff
@@ -0,0 +1,21 @@
+.section-title {
```

Comment:
I think it's more personal style, but unless I'm using SCSS I like to keep all the CSS in one file to take advantage of reusable properties. So my suggestion would be to just have one App.css file for readability and reusable code